ringpop.go

```diff
@@ -783,6 +783,15 @@ func (rp *Ringpop) Labels() (*swim.NodeLabels, error) {
 	return rp.node.Labels(), nil
 }
 
+func (rp *Ringpop) SetIdentity(identity string) error {
```

Comment:
Do we really want to support changing the identity after bootstrap? and why don't we support it before bootstrapping?